en/simulation/hitl.md

```diff
@@ -26,7 +28,35 @@ These parameter are optional and allow to run HITL without a radio remote contro
 
 ## Connect to the Simulator
 
-Currently either jMAVSim or X-Plane and QGroundControl are supported. Follow either setup step below.
+Currently Gazebo, jMAVSim, or X-Plane and QGroundControl are supported. Follow either setup step below.
```

Comment:
The docs don't say when/how you should connect the autopilot/vehicle to the computer. Does it matter?

Reply:
I think it does matter when you are first setting it up. I also forgot to add an important step for QGC configuration. I will add these steps in.